### DISCUSSION TITLE
what exactly should contain "path/to/your-avatar-style"

### DISCUSSION THREAD

**Discussion by pierroo** [dicebear/dicebear#370] [Q&A] [answered]
I followed the "create an avatar style with figma"
everything works well up until step 8, where it mentions 
import * as style from 'path/to/your-avatar-style';

But what exactly is this folder?
Is it the unzipped archive from step 7?
do we actually need all of it, including the node_modules etc?

**Comment by FlorianKoerner** [comment, marked as answer]
This depends on your setup. If you are using TypeScript, you need the files in the `src` folder. If you are using JavaScript, you will need the files in the `lib` folder.

You can rename the folder whatever you want and put it somewhere in your project. Then specify the path to that folder in the import.

**Reply by pierroo** [reply]
awesome, I will test that tomorrow thank you :)
